test/src/test/java/hudson/model/DirectoryBrowserSupportTest.java

```diff
@@ -919,6 +920,7 @@ public void junctionAndSymlink_outsideWorkspace_areNotAllowed_windowsJunction()
             List<String> entryNames = getListOfEntriesInDownloadedZip((UnexpectedPage) zipPage);
             assertThat(entryNames, contains("intermediateFolder/public2.key"));
         }
+        new Launcher.LocalLauncher(StreamTaskListener.fromStderr()).launch().cmds("cmd", "/c", "rmdir", "/s", "/q", j.jenkins.getRootDir().getAbsolutePath()).start().join();
```

Comment:
why this cleanup?  isn't this exactly what the test harness does (and if that can not clean up because the file is locked why would `rmdir` be able to?

Reply:
https://github.com/jenkinsci/jenkins-test-harness/pull/198#discussion_r501366464